models/comment.es6.js

```diff
@@ -1,36 +1,30 @@
 import Base from './base';
-import process from 'reddit-text-js';
 
 class Comment extends Base {
   _type = 'Comment';
 
   validators () {
-    const body = this.bodyValidator.bind(this);
+    const body_html = this.bodyHtmlValidator.bind(this);
```

Comment:
If I understand correctly (looking at https://github.com/reddit/node-api-client/blob/master/models/base.es6.js#L43), the validator names need to match the property names. In this case, the property name is the result of raw API data (as opposed to the previous `bodyHtml` field). Should we prefer to replace underscored API fields with camelCase fields in the model, if we're not doing any processing?